.github/workflows/tests.yml

```diff
@@ -0,0 +1,54 @@
+name: Tests
```

Comment:
Dropping the nightly tests?
<img width="423" alt="image" src="https://user-images.githubusercontent.com/291133/234612154-09167320-ed7c-4b49-b27c-4ac0bc504ab8.png">
Seems like they were identical to the on-commit tests (if they were even being run) so it's fine by me. If in the future we need to revive them, we can.

Reply:
They weren't being run.

The difference compared to the on-commit tests are:
```diff
- CHANNEL=stable
+ CHANNEL=nightly
```

I tried to make the workflow parameterized in this PR but it wasn't working. Agreed about bringing them back in the future. This is the same problem we have in SDKs: what version of algod should we be testing against?

Reply:
cc: @cce we've talked about this before, you might be curious. This is one way we pin versions between repositories. It means there is a gap where we don't know if the downstream tools are still compatible with algod.